build.sbt

```diff
@@ -1,3 +1,7 @@
+// While maven is throttling travis builds
+ThisBuild / resolvers := ("Google Maven Central Mirror".at(
```

Comment:
I haven't got used to this sbt syntax yet, but shouldn't this be `+=`?

Reply:
ah, it's at the end -- confusing

Reply:
I think that would put it at the end, i wanted it at the start

Reply:
I see, ok